boards/openmote-b/Makefile.include

```diff
@@ -9,27 +9,32 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
 # Set default flash tool
 export PROGRAMMER ?= cc2538-bsl
 
+FLASHFILE ?= $(HEXFILE)
```

Comment:
This changed from `BINFILE` to `HEXFILE` also for the `cc2538-bsl.py`.

Reply:
Yes it does and this is a huge improvement